doc/installation/system/inifile.rst

```diff
@@ -112,6 +112,14 @@ If ``PI_HSM_PADDING`` is not set, the default internal padding is used::
 .. warning:: The two paddings are not compatible. Data encrypted with one padding can not decrypted with the other padding. 
    So if you change the padding later during the cause of operation, it will not be possible to decrypt old data!
 
+``PREFERRED_LANGUAGE`` is a list in which the preferred languages can be defined. The browser's language settings are compared to this
```

Comment:
As discussed, this should be something like `PI_PREFERRED_LANGUAGE`

Reply:
fixed in f1975862ca4f27f3f4a5882c97848bc68a575245.